added key points (#12)
stemangiola/rpharma2020_tidytranscriptomics: vignettes/tidytranscriptomics.Rmd

### Perform analysis on subset and apply to full dataset

> Let's suppose we want to identify the markers that distinguish epithelial from endothelial cells, but we want to then visualising those genes across many cell types to understand the background transcription.

Reviewer: ```suggestion
Let's suppose we want to identify the markers that distinguish epithelial from endothelial cells, and we also want to then visualise the abundance of those transcripts across many cell types to understand their cell type specificity.
```

Reviewer: "Background transcription" is a bit unclear to me, what about something like this, if it makes sense to you?

Author: I'm not sure how to accept your suggestion.

Reviewer: There is a Commit Suggestion button, see here https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request

    custom_theme
```

## Key Points
- `tidybulk` allows streamlined multi-method analyses

> - The function `pivot_longer` from `tidyr` is useful to operate serially across consistent portions of data for comparative purposes. For example, the statistics for several methods. 

Reviewer: pivot_longer is not used in Part 2, it's used in Part 1. Do you mean to say nest from tidyr? Would it make sense instead to mention the map functions you're making use of from purrr?

Author: I meant pivot_ as it can help to format all methods and all statistcis at the same time, for then nesting it. and/or plotting with facets

Reviewer: ok but pivot_longer is not shown in Part 2? these should be the key points for what has been shown in this part so I think either need to show what you mean with pivot_longer in part 2 or drop it from the key points here.

Author: Ah ok :) happy to drop it

pbmc_small_nested_reanalysed
```

## Key Points
- Some basic steps of a single-cell RNA sequencing analysis are dimensionality reduction, cluster identification and cell type classification

> - `tidyseurat` is a invisible layer that operates on `Seurat` object and allows to visualise and manipulate data as if it were a tidy data frame.

Reviewer: ```suggestion
- `tidyseurat` is an invisible layer that operates on a `Seurat` object and enables us to visualise and manipulate data as if it were a tidy data frame.
```

> - `tidyseurat` object are `Seurat object` so they can be used with any `Seurat` compatible method

Reviewer: ```suggestion
- `tidyseurat` object is a `Seurat object` so it can be used with any `Seurat` compatible method
```

Author: Is not accepting

Reviewer: that is weird, are you able to add a message into the Commit suggestion dropdown like in the gif here: https://github.com/sindresorhus/refined-github/issues/2122#issuecomment-499011026
do you get a permission denied error like in that gif?

Author: I click on commit suggestion and the text does not appear to change. However feel free to accept everything and merge
